kubernetes/outreach.libsonnet

```diff
@@ -232,7 +236,7 @@ k + kubecfg {
         'alb.ingress.kubernetes.io/tags': 'cost=ingress_alb,outreach:environment=%s,kubernetesCluster=%s' % [cluster.environment, cluster.fqdn], 
         'alb.ingress.kubernetes.io/listen-ports': '[{"HTTP":80},{"HTTPS":443}]',
         'alb.ingress.kubernetes.io/actions.ssl-redirect': '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}', // Redirect http to https
-        'alb.ingress.kubernetes.io/ssl-policy': 'ELBSecurityPolicy-TLS-1-2-Ext-2018-06',
+        'alb.ingress.kubernetes.io/ssl-policy': if std.member(tls13, cluster.global_name) then 'ELBSecurityPolicy-TLS13-1-2-2021-06' else 'ELBSecurityPolicy-TLS-1-2-Ext-2018-06',
```

Comment:
Isn't cluster.global_name going to be <environment>.<region>? I think this is going to error.

https://github.com/getoutreach/jsonnet-libs/blob/ee7b4dd4fed3be2bbddb86dee34ffd3116ba0e2f/kubernetes/cluster.libsonnet#L5

Reply:
I changed it to `cluster.fqdn`